worlds/noita/docs/en_Noita.md

```diff
@@ -0,0 +1,60 @@
+# Noita
+
+## Where is the settings page?
+
+The [player settings page for this game](../player-settings) contains all the options you need to configure and export a
+config file.
+
+## What does randomization do to this game?
+
+Noita is a procedurally generated Roguelike. During runs in Noita you will find potions, wands, spells, perks and
+chests. For purposes of Archipelago multiworld functionality, collecting chests, results in an item check. Noita items 
+that can be found in other players games include, gold, max health increase, spell refresh, wands and perks. 
```

Comment:
It says perks can be found in the other games here but I don't see perks below in the list under "What Noita items can appear in other players' worlds?" nor do I see it in Items.py. Is there a reason it isn't there?

Could orbs also be included (as an option)? I'm not sure if other items should also be included or not (broken wand, eggs, the house thing that heals you, elemental rocks, etc) but I think the more unique items (instead of gold which I'd argue against), the more it will promote more versatility and communication with the other AP players (when it involves shops).

Reply:
Perks aren't yet implemented, I tried some testing with perks in the pool, and without making them extreamly rare ended up with a mega build super early and runs became trivial so decided to drop them, but I'm in two minds about that. This is all up for testing and tweaking, for future enhancements I also want to implement mini boss locations and get the location logic working.

I'll take another look at this tonight, I think you're right on the gold front, a more diverse set of items would make this more interesting.

Reply:
Noted. You're right and not all perks may be welcome.